src/components/Edit/ChangeSummary.vue

```diff
@@ -114,6 +115,11 @@ export default class ChangeSummary extends Mixins(
 
     return `${this.getBusinessNumber || '[Incorporation Number]'} B.C. Ltd.`
   }
+
+  /** The firm type label. */
+  get firmLabel (): string {
+    return this.getResource.changeData.orgPersonInfo.orgPersonLabel
```

Comment:
Is this chain of objects guaranteed to be declared (in store state model)?

Reply:
Yes, but in the store resource model.